### ISSUE TITLE
Feature Request: Ability to use ACF/Dynamic Fields in Query ID

### ISSUE THREAD

**Issue by flarewebdesign** [elementor/elementor#16725]
- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

### **What problem is your feature request going to solve? Please describe.**

Currently: Alice creates ACF template, using the post widget she uses a custom Query ID. For that ACF template, Alice would have to use the same query on every page.

What it will solve: Alice creates an ACF template while using the post widget she's now able to use different custom Query ID's dynamically. She can now select which category to automatically pull from, depending on the page.

![QueryID](https://user-images.githubusercontent.com/28833563/139101915-9e92e330-2ed4-44cf-8ac6-5a801364b57f.jpg)

### **Describe the solution you'd like**

Added functionality for dynamic fields for Query ID, this will help with custom post types.

### **Describe alternatives you've considered**

The closest alternative is Advanced Post Queries. Which hasn't been updated in 11 months.

### **Additional context**

In conclusion, adding the ability to use dynamic fields in the Query ID would allow for even more intricate use of Elementor.
